backend/tests/unit/routes/test_settings_routes.py

```
@@ -0,0 +1,238 @@
+# ---------------------------------------------------------------------------
+# Fake encryption helpers (no APP_SECRET_KEY needed)
+# ---------------------------------------------------------------------------
+
+
+def _mock_encrypt(value: dict) -> str:
+    return "ENC:" + json.dumps(value, sort_keys=True)
+
+
+def _mock_decrypt(value: str) -> dict:
+    if not value.startswith("ENC:"):
+        raise ValueError("Bad ciphertext")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)